turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -289,4 +289,69 @@ mod tests {
         assert_eq!(last_tick_height, expected_last_height);
         assert_eq!(res_entries, vec![last_entry]);
     }
+
+    #[test]
+    fn test_keep_coalescing_exact_boundary_exits() {
+        let typical = get_data_shred_bytes_per_batch_typical();
+        let mut stats = ProcessShredsStats::default();
+        let last_tick_height = 1;
+        let max_tick_height = 10;
```

Comment:
maybe we can make test local constants for `LAST_TICK_HEIGHT` and `MAX_TICK_HEIGHT` and re-use across these tests